docs/learning/08-phase1-layer-patterns.md

```
@@ -0,0 +1,260 @@
+}
+```
+
+`create()` 파라미터에 필수 필드가 모두 있으므로, 빠뜨리면 컴파일 단계에서 잡힌다.
+
+---
+
+## 4. Port vs 구현체 — 네이밍 규칙 적용 범위
+
+Port (out) 인터페이스는 비즈니스 언어로 명명한다는 규칙이 있다. 이 규칙은 **Port 인터페이스에만 적용**된다. 구현체 내부는 해당 없다.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)